pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "language-detect"
+dependencies = [
+    "joblib==1.4.2", 
+    "unicodedataplus==16.0.0", 
+    "fuzzywuzzy==0.18.0", 
+    "Levenshtein==0.26.1"
+]
+requires-python = ">=3.8"
+authors = [
+    {name = "Kavitha Raju", email = "Kavitha.Raju@bridgeconn.com"},
+]
+maintainers = [ 
+    {name = "BCS Team"}
+]
```

Comment:
Is this how we used to do? 
Its better to put author as "BCS Team" and in maintainers list add both of our names and emails. For the author, field also give a valid email id, either mine or joel@bridgeconn.com.